Src/AutoMoq/AutoMockPropertiesCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// A command that populates all public writable properties/fields of a mock object with anonymous values.
+    /// </summary>
+    public class AutoMockPropertiesCommand : ISpecimenCommand
```

Comment:
Yes, it can!

Please accept my sincere apologies that I have been sitting on this for so long! That's a poor experience for you, as a contributor! :flushed:

Thank you for nudging me to look at this again!